components/metal/CapabilityCard.vue

```diff
@@ -8,9 +8,22 @@
         <span v-html="description" />
       </div>
     </div>
+    <video
+      v-if="isVideo()"
+      class="capability-card__visual-resource"
+      loop
+      autoplay
+      muted
+      playsinline
+    >
```

Comment:
@korgan00 - this is something we might have not discussed during design handoff, but for accessibility reasons, I think we need to enable the `controls` attribute, at minimum.

Realistically, and we can do this in a separate issue (which I'll file after), we could be better with our accessible approach by incorporating [custom controls](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia).

I just didn't want to block this PR cause what you've built is still within scope.

Reply:
Ok, thank you, that is interesting.